Packs/VulnDB/Integrations/VulnDB/README.md

```diff
@@ -1572,4 +1572,50 @@
 <p> </p>
 <h5>
 <a id="Command_Example_486"></a>Command Example</h5>
-<pre>!vulndb-get-vuln-by-cve-id cve="2013-1228" max_size="20"</pre>
\ No newline at end of file
+<pre>!vulndb-get-vuln-by-cve-id cve="2013-1228" max_size="20"</pre>
+### vulndb-get-cpe-by-vuln-id
+
+***
+Returns the CPE(s) for a vulnerability, by VulnDB ID.
+
+#### Base Command
+
+`vulndb-get-cpe-by-vuln-id`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| vuln_id | VulnDB vulnerability ID. | Required | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| VulnDB.CPE.Value | string | The CPE\(s\) for the given vulnerability. | 
+
+### vulndb-get-vuln-report-by-vuln-id
+
+***
+Returns the PDF report for a vulnerability as generated by VulnDB.
+
+#### Base Command
+
+`vulndb-get-vuln-report-by-vuln-id`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| vuln_id | ID of the vulnerability for which to return information. | Required | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| InfoFile.Name | unknown | Name of the report. | 
+| InfoFile.EntryID | unknown | Entry ID of the report. | 
+| InfoFile.Size | unknown | Report size. | 
+| InfoFile.Type | unknown | Report type e.g. "PDF". | 
```

Comment:
```suggestion
| InfoFile.Type | unknown | Report type, e.g., "PDF". | 
```

Reply:
It will not change because it is not related to the above contribution but to the commit from master.